docs/developer_guide/testing_suite.rst

```diff
@@ -0,0 +1,63 @@
+Running GIN tests locally
+=========================
+
+`neuroconv` verifies the integrity of all code changes by running a full test suite on short examples of real data from the formats we support.
+
+The testing suite is broken up into sub-folders based on the scope of functionalities and dependencies you wish to test.
+
+We recommend always running tests in a fresh environment to ensure errors are not the result of contaminated dependencies.
+
+There are three broad classes of tests in this regard...
+
+
+
+Minimal
+-------
+
+These test internal functionality using only minimal dependencies or pre-downloaded data.
+
+Subfolders: `tests/test_minimal <https://github.com/catalystneuro/neuroconv/tree/main/tests/test_minimal>`_ and `tests/test_internals <https://github.com/catalystneuro/neuroconv/tree/main/tests/test_internals>`_
```

Comment:
Using this type of link ensures that our link checker CI will remind us to update the links here if we end up changing the testing folder structure or names in any way